.github/workflows/release.yml

```diff
@@ -116,3 +116,21 @@ jobs:
           exit-code: '1'
           ignore-unfixed: true
           severity: 'CRITICAL,HIGH'
+
+  downstream-tagging:
+    runs-on: ubuntu-latest
+    # since trivy already depends on linux-utils, its not included here
+    needs: ['trivy']
+    steps:
+      - name: Downstream tagging
+        uses: akhilerm@openebs-release-mgmt@main
+        with:
+          tag-name: ${{ github.ref }}
+          body: 'Release created from linux-utils'
+          repo: |
```

Comment:
Another check we should decide is, if the release tagging one repo fails, should we proceed to the next or stop.
a `fail-fast:true` can be set to check for this. cc: @kmova

Reply:
It is possible that the failure is due to the tag already being present, which should be treated idempotent and shouldn't fail. 

In case of other failures - makes sense to proceed - just in case the failure is due to a branch missing on one of the downstream repo.  After proceeding, can the status of this job be marked fail if any one of the tagging has failed.

Reply:
> which should be treated idempotent and shouldn't fail.

It has to be changed. Currently if we do that, the status code is 422, and we are checking only for 201.

Reply:
> After proceeding, can the status of this job be marked fail if any one of the tagging has failed.

yes

Reply:
> In case of other failures - makes sense to proceed - just in case the failure is due to a branch missing on one of the downstream repo.

Should this be configurable?

This is what I understood from the comments:
1. the tagging should be idempotent, if tag is already present, it is a success
2. if the tagging fails due to some other error, 2 cases:
      -  `fail-fast:true`  : tagging of further repos will fail
      - `fail-fast:false`  : tagging of further repos will continue
In both the cases, the status of action will be `failure()`

@kmova can you confirm on this?

Reply:
above is correct @akhilerm

Reply:
Done with the changes @kmova